support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/RegistryConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package org.apereo.cas.config;
+
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.dao.CassandraDao;
+import org.apereo.cas.dao.NoSqlTicketRegistry;
+import org.apereo.cas.dao.NoSqlTicketRegistryDao;
+import org.apereo.cas.logout.LogoutManager;
+import org.apereo.cas.serializer.JacksonJSONSerializer;
+import org.apereo.cas.ticket.registry.TicketRegistry;
+import org.apereo.cas.ticket.registry.TicketRegistryCleaner;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@Configuration("ticketRegistryConfiguration")
```

Comment:
Update the name of the configuration. This deals with Cassandra; not a generic ticket registry config.